src/hexkit/protocols/objstorage.py

```diff
@@ -220,6 +228,7 @@ async def copy_object(
         source_object_id: str,
         dest_bucket_id: str,
         dest_object_id: str,
+        abort_failed: bool = True,
```

Comment:
Can you mention this argument in the docstring?

Reply:
Added some explanation, including when this should be set to `False` in 0fa7eb3935636c9349c1609fbba698d1558b5417